### PR TITLE
Fix missing comma in connection options

### PR DESCRIPTION
**Pull request by SaitTalhaNisanci**
https://github.com/citusdata/citus/commit/a31429aae5aa6c8271ec3dee663d70a67f64e60d#r55402663

### CODE REVIEW

File: src/backend/distributed/shared_library_init.c [hunk 1]
```diff
@@ -1895,7 +1895,7 @@ NodeConninfoGucCheckHook(char **newval, void **extra, GucSource source)
 		"sslcompression",
 		"sslcrl",
 		"sslmode",
-		"sslrootcert"
+		"sslrootcert",
 		"tcp_user_timeout",
```
Comment (marcocitus) [inline src/backend/distributed/shared_library_init.c:1899]: should the final comma be removed?

File: src/backend/distributed/shared_library_init.c [hunk 2]
```diff
@@ -1895,7 +1895,7 @@ NodeConninfoGucCheckHook(char **newval, void **extra, GucSource source)
 		"sslcompression",
 		"sslcrl",
 		"sslmode",
-		"sslrootcert"
+		"sslrootcert",
```
Comment (marcocitus) [inline src/backend/distributed/shared_library_init.c:1898]: Good catch. C is mean.

File: src/backend/distributed/shared_library_init.c (hunk 1 above)
Comment (SaitTalhaNisanci) [reply src/backend/distributed/shared_library_init.c:1899]: both of them seems to be working, as in we can set the `tcp_user_timeout` variable with a comma at the end or without.

Comment (marcocitus) [reply src/backend/distributed/shared_library_init.c:1899]: Some older compilers might complain, but perhaps they've already gone extinct.

**Review by marcocitus — APPROVED** [review]

**Pull request merged**